alma/alma-woocommerce-gateway
# Feat/v4.2.5

## Reason for change

- fix : css issues on checkout accordion depending the theme

https://linear.app/almapay/issue/MPP-273/fix-template-checkout-issue

### How to test

- install different free themes to see if the checkout is still with an accordion



### Checklist for authors and reviewers




### Non applicable
